api/geth_backend.go

```diff
@@ -2658,6 +2658,17 @@ func (b *GethStatusBackend) injectAccountsIntoWakuService(w types.WakuKeyManager
 		b.statusNode.ChatService(accDB).Init(messenger)
 		b.statusNode.EnsService().Init(messenger.SyncEnsNamesWithDispatchMessage)
 		b.statusNode.CommunityTokensService().Init(messenger)
+
+		if messenger != nil && b.statusNode.WakuV2Service() != nil {
+			var deviceType string
+			for _, inst := range messenger.Installations() {
+				if inst.ID == messenger.InstallationID() {
+					deviceType = inst.InstallationMetadata.DeviceType
+					break
+				}
+			}
+			b.statusNode.WakuV2Service().InitTelemetryClient(deviceType)
```

Comment:
Why do we need to set `deviceType` through waku service? 🤔 
I think it would be cleaner to `Messenger.TelemetryService().SetDeviceType()`?

Reply:
Or even better, just call `m.telemetryClient.SetDeviceType()` inside `InitInstallations`:
https://github.com/status-im/status-go/blob/f123e98179d9669a5a25e2e7819b4b257e5969ba/protocol/messenger_pairing_and_syncing.go#L403-L419

Reply:
Updated in last commit